web_app.py

```
@@ -1157,14 +1234,27 @@ def _army_belongs_to_region(self, army: Dict[str, Any], region: Dict[str, Any])
         )
 
     def _army_belongs_to_theater(self, army: Dict[str, Any], theater_id: str) -> bool:
+        """theater 归属。禁裸「辽东」扫全字段——否则 辽东侧翼/门户/外线 误挂 liaodong，
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)